### ISSUE TITLE
Unable to build DynamoDB

### ISSUE THREAD

**Issue by rinosh1989** [mathworks-ref-arch/matlab-aws-dynamodb#1]
Hi, I encountered the following error.
<img width="591" alt="Screenshot" src="https://user-images.githubusercontent.com/7686819/62353822-20cb6d00-b529-11e9-83ec-33e3902abcac.png">
Following are the steps followed

1. Installed Java JDK - 12.0.2 64bit and added `JAVA_HOME` to environment variables.
1. Installed Maven 3.6.1 and added the bin location to `PATH` environment variable.
1. Git cloned the mathworks-aws-support project.
1. Tried to install AWS SDK for Java 2.0 but failed.
1. moved into mathworks-aws-support/matlab-aws-dynamodb/Software/Java directory and ran the command `mvn clean package` and I got the following error
```
Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project aws-sdk: Compilation failure: Compilation failure:
[ERROR] Source option 5 is no longer supported. Use 7 or later.
[ERROR] Target option 5 is no longer supported. Use 7 or later.
```

**Comment by brownemi** [comment]
Hi rinosh1989,

Thanks for your feedback and bringing this to our attention.
In the short-term please amend the final lines of the dynamodb pom.xml file to include the properties section so that it look likes the following, and rerun the ```mvn clean package``` step, or build with an older JDK e.g. 8. A future release will address this issue.

```xml
  </build>
     <properties>
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
</project>
```

**Comment by hosagrahara** [comment]
> A future release will address this issue.

This was fixed in fa7e0aec957ff12e4416a018bdca547d515c9596
